src/api/modelTypes.js

```diff
@@ -643,6 +643,11 @@ export type Topic = $ReadOnly<{|
 export enum UserTopicVisibilityPolicy {
   None = 0,
   Muted = 1,
+  // Not in the API docs yet, but the API has been agreed on:
+  //   https://chat.zulip.org/#narrow/stream/378-api-design/topic/muted.20topics/near/1501574
+  //   https://chat.zulip.org/#narrow/stream/378-api-design/topic/muted.20topics/near/1528885
+  // TODO(server): delete this comment once documented
+  Unmuted = 2,
```

Comment:
I do see this in the API doc: https://zulip.com/api/update-user-topic

Reply:
Hmm true. It doesn't appear in the other places this enum is used, though:
https://zulip.com/api/register-queue
https://zulip.com/api/get-events#user_topic

I guess I should raise this in `#api documentation`.

Reply:
https://chat.zulip.org/#narrow/stream/378-api-design/topic/muted.20topics/near/1557615

(Mentioned in `#api design` instead because there was a previous message there to follow up on.)